tests/commands/help.spec.ts

```diff
@@ -30,7 +30,6 @@ PRIMARY COMMANDS
   push <applicationOrDevice>             start a remote build on the balenaCloud build servers or a local mode device
   logs <device>                          show device logs
   ssh <applicationOrDevice> [service]    SSH into the host or application container of a device
-  apps                                   list all applications
```

Comment:
left over I think?

Reply:
It was leftover from before, we don't have an `apps` command anymore, guess we just never removed it from here

I decided to squeeze it in instead of giving it its own PR :P